src/Microsoft.ManagedZLib/benchmarks/ManagedZLibBenchmarks.cs

```diff
@@ -62,15 +63,25 @@ public void DecompressManaged()
     {
         CompressedFile!.CompressedDataStream.Position = 0;
         _outputStream!.Position = 0;
-
-
```

Comment:
`decompressor` below should also be disposed, which also means it should be created with leaveOpen: true.

Same in DecompressNative above.

And same in CompressedFile's ctor.

Reply:
I'll add the leaveOpen parameter, but I had the doubt: If the instantiation (there's no global variable) of the decompressor is done inside the context of the benchmark, wouldn't it be disposed anyways at the end?

Reply:
No, there's no implicit destruction at the end of a scope as in C++. That's what `using` is for.

Reply:
I didn't want to add overhead for measuring the algorithm's work, but I think I'll do 2 different benchmark's categories for it: One with instantiation+dispose and one just with the copyTo.